README.md

```diff
@@ -24,6 +24,7 @@ Just invoke *logme* in your Fastfile action. Logme will return the commit messag
 | `to_revision` | End revision commit | false |
 | `from_revision` | From revision commit | false |
 | `message_regexp_filters` | Regexp filters like. ie '^MDM-\|^CTS:' | true |
+| `message_regexp_exclude_filters` | Regexp filters like. ie '^MDM-TST\|^CTS:UI' | true |
```

Comment:
Isn't this functionality achieved with a single regex filter? 🤔 

Additionally, I think that having two regex filters may create confusion on the caller as to how they interact. For instance, is the include filter run first and then the exclude, or vice versa?